docs/codeql/codeql-cli/analyzing-databases-with-the-codeql-cli.rst

```diff
@@ -105,13 +111,23 @@ see ":doc:`Using custom queries with the CodeQL CLI <using-custom-queries-with-t
 Running GitHub code scanning suites
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The CodeQL repository also includes query suites, which can be run over your
-code as part of a broader code review. CodeQL query suites are ``.qls`` files
-that use directives to select queries to run based on certain metadata
-properties.
+To run the GitHub code scanning suite of queries over a CodeQL datbase for a C/C++ codebase, 
+you could use the following command from the directory containing your database::
+
+   codeql database analyze <cpp-database> cpp-code-scanning.qls --format=sarifv2.1.0 --output=cpp-analysis/cpp-results.sarif
```

Comment:
Minor: Could remove the directory for simplicity. Otherwise this will fail if the user hasn't already created the `cpp-analysis` directory.
```suggestion
   codeql database analyze <cpp-database> cpp-code-scanning.qls --format=sarifv2.1.0 --output=cpp-results.sarif
```